### DISCUSSION TITLE
How to install "bun:test" at project level

### DISCUSSION THREAD

**Discussion by Christophvh** [oven-sh/bun#9277] [Q&A]
Hi,

I want to use "bun:test" instead of Jest or Vitest. But I can't find any documentation on how to 'only' install buntest on a project level. 

Both Jest and Vitest offer project-based installs. 

I would like to avoid installing bun on global level. This is very cumbersome for other developers in the team or when using CI/CD.

**Comment by Kenzo-Wada** [comment]
I suggest proto for package manager manager (like nvm, volta) 

you can pin the versions for the project level.

https://moonrepo.dev/proto
